lib/adapters/file/read.js

```diff
@@ -9,7 +9,7 @@ var Read = Juttle.proc.source.extend({
     procName: 'read-file',
 
     initialize: function(options, params, location, program) {
-        var allowed_options = ['from', 'to', 'file', 'timeField', 'format'];
+        var allowed_options = ['from', 'to', 'file', 'timeField', 'format', 'formatOptions'];
```

Comment:
Using `-formatOptions` seems opaque. I'd lean more towards adding a `-pattern` option that's only valid when reading with format grok.